### ISSUE TITLE
[Feature] CMS Dashboard User Traffic and Main Game Dashboard

### ISSUE THREAD

**Issue by xHayden** [GTBitsOfGood/jennifer-anns-group#159]
## Context
We're starting to implement the CMS Dashboard pages. All that's created so far is a Provider for handling the authentication with the analytics API, and an event is created whenever anyone goes to a page.
We need to implement the frontend for the CMS dashboard following the Figma.

### Frontend
- Create User Traffic and main game dashboard as displayed on the CMS figma
- Leaderboard and Game Info is __not__ a part of this ticket
- Use [Nivo](https://nivo.rocks/) to create the pie charts that are display
- Don't implement the Links Page, it will use the same pagination as another ticket.

### Backend
- Fetch data from bog analytics custom visit events
- Populate user traffic sources with data coming from a remote url (referrer is remote)
- Populate game traffic with user group roles from the same data

## Acceptance Criteria
- [ ] Create User Traffic and Main Game Dashboard
- [ ] Use Nivo to create graphs
- [ ] Fetch bog analytics custom visit data
- [ ] Use visit data to populate graphs

## Notes
It doesn't matter where you put these components for now, I'll move them to the correct location. Another ticket was made for the main views.
[Dashboard on Figma](https://www.figma.com/design/mgIynQl7FOJ10aAEdBy5u6/Jennifer-Annes-F24?node-id=15879-1582&t=BarmXPBvJ1dLtAIX-1)
[Bog Analytics NPM Package Docs](https://github.com/GTBitsOfGood/bog-analytics/tree/main/package#bits-of-good-analytics-npm-package)

Depends on #158 to be completed. If unable to get event data, create test data to demo the functionality.
